dev-docs/bidders/beop.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: bidder
+title: BeOp
+description: BeOp Bidder Adaptor
+pbjs: true
+biddercode: beop
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Description | Example | Type     |
+|---------------|----------|-------------|---------|----------|
+| `accountId` or `networkId` | required |  Your BeOp account ID   | `'5a8af500c9e77c00017e4cad'`   | `string` |
+| `currency`      | optional |  Your currency        |  `'EUR'` (default) or `'USD'`   | `string` |
```

Comment:
For the record you should be primarily asking the publisher for their currency as a bidder-specific param. See the [Currency module](https://docs.prebid.org/dev-docs/modules/currency.html) for the standard location for pubs to define their currency.

Instead of 
```
  const publisherCurrency = utils.getValue(bid.params, 'currency') || 'EUR';
```
You should be doing something like this:
```
  const publisherCurrency = config.getConfig('currency.adServerCurrency') || utils.getValue(bid.params, 'currency') || 'EUR';
```
First look in the standard location, then it's ok to have a bidder-specific fall back param and an overall default.

Since your adapter has already been released, I'll go ahead and merge these docs, but am going to open a PBJS issue to track this resolution.

Reply:
Thanks! I'll create a PR right away